modules/ROOT/pages/type-definitions/directives/database-mapping.adoc

```diff
@@ -202,3 +203,44 @@ await startStandaloneServer(server, {
     context: async ({ req }) => ({ req, appId: "myApp" }),
 });
 ----
+
+
+[[type-definitions-alias]]
+== `@alias`
+
+This directive maps a GraphQL field to a Neo4j property on a node or relationship.
+It can be used on any fields that are not `@cypher` or `@relationship` fields.
+
+For example:
+
+[source, graphql, indent=0]
+----
+type User {
+    id: ID! @id @alias(property: "dbId")
+    username: String!
+}
+----
+
+[source, graphql, indent=0]
+----
+type User {
+    id: ID! @id
+    username: String! @alias(property: "dbUserName")
+    livesIn: [City!]! @relationship(direction: OUT, type: "LIVES_IN", properties: "UserLivesInProperties")
+}
+
+type City {
+    name: String
+}
+
+type UserLivesInProperties @relationshipProperties {
+    since: DateTime @alias(property: "moveInDate")
+}
+----
+
+Note that the property in aliases are automatically escaped (wrapped with backticks ``), so there is no need to add escape characters around them.
```

Comment:
Maybe turn this into a note? I guess it might look a bit too heavy with two boxes (the code and then the note), but I think this is important information to be highlighted